app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v4.rb

```diff
@@ -321,6 +321,24 @@ def uuid_from_href(ems_ref)
       URI(ems_ref).path.split('/').last
     end
 
+    def event_fetcher
+      EventFetcher.new(ext_management_system)
+    end
+
+    class EventFetcher
+      attr_reader :ext_management_system
+
+      def initialize(ems)
+        @ext_management_system = ems
+      end
+
+      def events(query_options)
```

Comment:
For this particular method the query parameters are the same in V3 and V4. However the `query` parameter is intended as mechanism to be able to work with parameters that the SDK doesn't support yet. That is not the case. I'd suggest to avoid using it here. Can we use directly the `max` and `from` parameters?